apps/test/unit/templates/manageStudents/ManageStudentsTableTest.js

```diff
@@ -274,6 +284,18 @@ describe('ManageStudentsTable', () => {
       expect(loginInfo.find('SignInInstructions').props().loginType).to.equal(
         SectionLoginType.clever
       );
+      expect(wrapper.containsMatchingElement(<NoSectionCodeDialog />)).to.equal(
+        true
+      );
+      expect(
+        wrapper.find('NoSectionCodeDialog').props().typeClassroom
+      ).to.equal(SectionLoginType.clever);
+      expect(wrapper.find('NoSectionCodeDialog').props().isOpen).to.equal(
```

Comment:
NIT (fyi) A shorthand for `to.equal(false)` is `to.be.false` in case you find yourself wanting to save a couple keystrokes :)